crates/topos-core/src/types/stream.rs

```diff
@@ -0,0 +1,133 @@
+use std::{fmt, ops::Deref};
+
+use serde::{Deserialize, Serialize};
+use thiserror::Error;
+
+use topos_uci::SubnetId;
+
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
+pub struct CertificateSourceStreamPosition {
```

Comment:
```suggestion
/// Contains the position of a certificate for a subnet.
#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
pub struct CertificateSourceStreamPosition {
```

…but I think that's not accurate. Can you help?

Reply:
Why not accurate?